bundles/config/org.eclipse.smarthome.config.discovery/src/main/java/org/eclipse/smarthome/config/discovery/internal/AutomaticInboxProcessor.java

```diff
@@ -92,7 +92,8 @@ public void thingAdded(Inbox inbox, DiscoveryResult result) {
 
     private String getRepresentationValue(DiscoveryResult result) {
         return result.getRepresentationProperty() != null
-                ? (String) result.getProperties().get(result.getRepresentationProperty()) : null;
+                ? (String) result.getProperties().get(result.getRepresentationProperty())
```

Comment:
I would be interested as well. Just double checked my settings of a brand new freshly installed Eclipse Oxygen using the Eclipse SmartHome setup - it says line braks at column 120 - still this formatting got changed somehow. Not sure why. May it be, that Oxygen has NEW formatting settings, appling them for ? : constructs?